code-review/README.md

```diff
@@ -20,8 +20,9 @@ Everyone
 * Don't use sarcasm.
 * Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
   If they are, use them with aplomb.
-* Talk in person if there are too many "I didn't understand" or "Alternative
-  solution:" comments. Post a follow-up comment summarizing offline discussion.
+* Talk in chat, screenshare, or in person if there are too many "I didn't 
```

Comment:
As a sometimes remote worker / remote advocate, I like to put the remote options first to level the perceived value. I think no matter the order, people will self select what makes the most sense for them. Even when I'm in office I often do a quick pair with Screenhero to walk people through code.